HinglishULMFiT_AWD_QRNN.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Do we have another experiment like this with Stanza tokenizer instead of <code>spm</code>?</p><p> </p><p>Just asking out of curiosity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Nope, we didn't try that.

Reply:
Cool, let's mark that for later then.